BitToGithub/Program.cs

```diff
@@ -61,7 +67,12 @@ static void Main(string[] args)
                                 Console.WriteLine("found issue {0}", issue.local_id);
                                 var newIssue = new NewIssue(issue.title);
                                 newIssue.Labels.Add(migratedLabel);
-                                newIssue.Body = issue.content + "\{Environment.NewLine}\{Environment.NewLine}> Originally created at \{issue.utc_created_on} (UTC) by \{issue.reported_by.username} as a(n) \{issue.priority} issue.";
```

Comment:
Nice, thanks for removing these. I believe these are pre-release way to have Interpolated Strings which has changed. 

You can use https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/interpolated-strings if you prefer to.

Reply:
I spend a lot of time being forced to code for CLR2 compatibility, so often miss out on all the fancy new sugar.